jest.config.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  coverageDirectory: './coverage/',
+  collectCoverage: true,
+  collectCoverageFrom: [
+    '<rootDir>/{components,packages}/*/src/**.[jt]s?(x)',
+    '<rootDir>/{components,packages}/*/src/**/*.[jt]s?(x)',
```

Comment:
in principle this line shouldn't be needed - as there's a `**` in the line above that's effectively the same as this one

Reply:
(`**` means "match any character, including `/`")